frontend/pipeline.yml

```diff
@@ -30,6 +30,21 @@ test:frontend:lint:
   tags:
     - hetzner-amd-beefy
 
+test:frontend:license-headers:
+  image: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/denoland/deno:debian-2.0.2
+  stage: test
+  rules:
```

Comment:
The runner tag is missing, I fear

Reply:
That gets inherited from the defaults in the root gitlab - so it runs on there already 👌 :
<img width="275" alt="Screenshot 2024-10-23 at 15 20 29" src="https://github.com/user-attachments/assets/6a9f29b7-cdfc-4c00-ba4b-839ed2826acc">